website/docs/r/pi_network_port_attach.html.markdown

```diff
@@ -8,6 +8,7 @@ description: |-
 ---
 
 # ibm_pi_network_port_attach
```

Comment:
This is the wrong documentation file for this resource. The file you are looking for is pi_cloud_connection_network_attach.html.markdown.